content/en/network_monitoring/devices/getting_started.md

```diff
@@ -0,0 +1,132 @@
+
+[Autodiscovery][3]
+: Configure SNMP monitoring using Autodiscovery.
+
+[Ping][13]
+: Additionally, SNMP supports enabling ping on your devices.
+
+## Phase 3: Additional customizations (optional)
+
+### SD-WAN monitoring
+
+Alongside SNMP devices, you can monitor wireless and SD-WAN (Software-Defined Wide Area Network) environments for select vendors, such as metrics from wireless access points, and the health of SD-WAN tunnels and edge devices.
+
+SD-WAN is a type of networking technology that uses software-defined networking (SDN) principles to manage and optimize the performance of wide area networks (WANs). It is mainly used to interconnect remote offices and data centers across different transports (MPLS, Broadband, 5G, and so on). SD-WAN benefits from automatic load balancing and failure detection across these transports. 
+
+Datadog supports the following vendors for SD-WAN network monitoring:
+
+  - [Meraki SD-WAN][15] (public beta)
+  - [Cisco SD-WAN][14] (public beta)
+
+### Enrich network devices with tags 
+
+Once NDM is configured on your devices, you can further enrich them by adding network device tags using the following methods:
+
+[Datadog Agent][1]
+: The Agent can collect device tags when configuring [individual devices][2] or with [Autodiscovery][3].
+
+[Device profiles][16]
+: Configure the Agent to collect and customize specific metrics and tags by creating Device profiles directly in the app.
+
+[ServiceNow integration][4]
+: Dynamically enrich network devices monitored by Datadog Network Device Monitoring with data defined in ServiceNow's CMDB (Configuration Management Database).
+
+[Network Device Monitoring API][5]
+: Utilize the Network Device Monitoring API to programmatically add tags to your network devices.
+
+### Customize metrics and tags
+
+Customize metrics and tags on your devices by viewing the [Supported Devices][22] page to view our out-of-the-box device profiles. If you would like to edit/add more metrics, the following options are available:
+
+[Device profiles][23]
+: Directly edit metrics and tags in the Datadog Agent `yaml` file with with device profiles.
+
+[GUI based profile authoring][24]
+: Take advantage of Datadog Network Monitoring's GUI based device onboarding experience where you can add custom metrics and tags to your devices.
+
+### NetFlow Monitoring 
+
+Configure [NetFlow Monitoring][21] to visualize and monitor your flow records from your NetFlow-enabled devices.
+
+{{< img src="network_device_monitoring/netflow/home.png" alt="The NetFlow Monitoring page containing tabs for top sources, destinations, protocols, source ports, destination ports, and device trends" style="width:100%;" >}}
+
+## Phase 4: Validate your data
+
+- Start monitoring your entire network infrastructure on the [Network Devices][17] page.
+- View metrics collected on Datadog's out-of-the-box dashboards:
+  - [Overview of all devices monitored][8]
+  - [Across the performance on all interfaces][18]
```

Comment:
Was this line originally part of a sentence with the previous item? As a list item, it's a bit confusing to me because it seems to describe something but not have a clear subject. Perhaps "Performance of all interfaces on your network devices" might be clearer?